app/src/main/res/layout/activity_about.xml

```diff
@@ -36,22 +38,30 @@
                 android:id="@+id/textView2"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:autoSizeTextType="uniform"
+                app:layout_constraintWidth_percent="0.3"
+                android:lines="1"
                 android:text="@string/app_name"
                 android:textColor="@color/white"
-                android:textSize="30sp"
+                android:textSize="30dp"
+                android:textAlignment="center"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintStart_toEndOf="@+id/imageView3"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintHeight_percent="0.4"
+                app:layout_constraintLeft_toRightOf="@+id/imageView3"
+                app:layout_constraintRight_toLeftOf="@+id/appVersion"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintHorizontal_bias="0.1" />
 
             <TextView
                 android:id="@+id/appVersion"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:paddingStart="15dp"
```

Comment:
Please add some margin here

Reply:
OK! I will work on this right now. :)

Reply:
Hello! After some tries, I just found out that maybe the padding of "10dp" would be more suitable. Like this one:

![Screenshot_1630293300](https://user-images.githubusercontent.com/25502419/131280267-ca416447-b5e7-479a-b6c3-c99b0bb76781.png)

So would you agree with this one if it is OK to you? :) Thanks!